test/abilities/neutralizing-gas.test.ts

```diff
@@ -0,0 +1,197 @@
+    const enemyPokemon = game.field.getEnemyPokemon();
+    expect(enemyPokemon.getStatStage(Stat.ATK)).toBe(-1);
+
+    game.move.select(MoveId.SPLASH);
+    await game.toEndOfTurn();
+    // Intimidate did not reactivate after neutralizing gas was removed
+    expect(enemyPokemon.getStatStage(Stat.ATK)).toBe(-1);
+  });
+
+  it("should only deactivate when all setters are off the field", async () => {
+    game.override.enemyMoveset([MoveId.ENTRAINMENT, MoveId.SPLASH]).battleType("double");
+    await game.classicMode.startBattle(SpeciesId.ACCELGOR, SpeciesId.ACCELGOR);
+
+    game.move.select(MoveId.SPLASH, 0);
+    game.move.select(MoveId.SPLASH, 1);
+    game.setTurnOrder([BattlerIndex.PLAYER, BattlerIndex.PLAYER_2, BattlerIndex.ENEMY, BattlerIndex.ENEMY_2]);
+    await game.move.selectEnemyMove(MoveId.ENTRAINMENT, BattlerIndex.PLAYER);
+    await game.move.selectEnemyMove(MoveId.SPLASH);
+    await game.toEndOfTurn();
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeDefined(); // Now one neut gas user is left
+
+    game.move.select(MoveId.SPLASH, 0);
+    game.move.select(MoveId.SPLASH, 1);
+    game.setTurnOrder([BattlerIndex.PLAYER, BattlerIndex.PLAYER_2, BattlerIndex.ENEMY, BattlerIndex.ENEMY_2]);
+    await game.move.selectEnemyMove(MoveId.ENTRAINMENT, BattlerIndex.PLAYER_2);
+    await game.move.selectEnemyMove(MoveId.SPLASH);
+    await game.toEndOfTurn();
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeUndefined(); // No neut gas users are left
+  });
+
+  it("should deactivate when suppressed by gastro acid", async () => {
+    game.override.enemyMoveset(MoveId.GASTRO_ACID);
+
+    await game.classicMode.startBattle(SpeciesId.FEEBAS);
+
+    game.move.select(MoveId.SPLASH);
+    await game.toEndOfTurn();
+
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeUndefined();
+  });
+
+  it("should deactivate when the pokemon faints", async () => {
+    game.override.ability(AbilityId.BALL_FETCH).enemyAbility(AbilityId.NEUTRALIZING_GAS);
+
+    await game.classicMode.startBattle(SpeciesId.FEEBAS);
+    game.move.select(MoveId.SPLASH);
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeDefined();
+    // await game.doKillOpponents();
+
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeUndefined();
+  });
+
+  it("should deactivate upon catching a wild pokemon", async () => {
+    game.override.battleType("single").enemyAbility(AbilityId.NEUTRALIZING_GAS).ability(AbilityId.BALL_FETCH);
+    await game.classicMode.startBattle(SpeciesId.MAGIKARP);
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeDefined();
+
+    game.scene.pokeballCounts[PokeballType.MASTER_BALL] = 1;
+    // game.doThrowPokeball(PokeballType.MASTER_BALL);
+    await game.toEndOfTurn();
+
+    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeUndefined();
+  });
```

Comment:
A test without any `expect` just eats time and resources.
Should probably mark it as `skip` or `todo`.

```suggestion
  it.skip("should deactivate upon catching a wild pokemon", async () => {
    game.override.battleType("single").enemyAbility(AbilityId.NEUTRALIZING_GAS).ability(AbilityId.BALL_FETCH);
    await game.classicMode.startBattle(SpeciesId.MAGIKARP);
    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeDefined();

    game.scene.pokeballCounts[PokeballType.MASTER_BALL] = 1;
    // game.doThrowPokeball(PokeballType.MASTER_BALL);
    await game.toEndOfTurn();

    // expect(game.scene.arena.getTag(ArenaTagType.NEUTRALIZING_GAS)).toBeUndefined();
  });
```